PluginBuilder/Controllers/HomeController.cs

```diff
@@ -170,7 +176,22 @@ public async Task<IActionResult> CreatePlugin(CreatePluginViewModel model)
                 ModelState.AddModelError(nameof(model.PluginSlug), "This slug already exists");
                 return View(model);
             }
+
+            if (model.Logo != null)
+            {
+                try
```

Comment:
You could add the image file checks like we have them on the server here, so that we validate it's an image and below a maximum file size.